routers/repo/http.go

```diff
@@ -276,6 +277,19 @@ func HTTP(ctx *context.Context) {
 		}
 	}
 
+	if !repoExist {
+		repo, err = models.CreateRepository(authUser, authUser, models.CreateRepoOptions{
```

Comment:
Should this only be possible for user repositories, what about organizations? At least there need to be some check.

Reply:
There is now a config for both users and orgs for flexibility.